android/app/src/main/java/com/mattermost/components/CustomTextInput.java

```diff
@@ -1,30 +0,0 @@
-package com.mattermost.components;
-
-import android.content.Context;
-import android.text.InputType;
-
-import com.facebook.react.views.textinput.ReactEditText;
-
-public class CustomTextInput extends ReactEditText {
```

Comment:
Is this related to this fix - [On Android, TextInput now takes onKeyPress like iOS](https://github.com/facebook/react-native/commit/c9ff0bc)?

Reply:
nope, is related to [Fixed autoscroll to cursor on Android TextInput](https://github.com/facebook/react-native/commit/0bef872)